src/cat.sol

```diff
@@ -62,9 +62,9 @@ contract Cat is LibNote {
 
     mapping (bytes32 => Ilk) public ilks;
 
-    uint256 public live;
-    VatLike public vat;
-    VowLike public vow;
+    uint256 public live;  // Shutdown Flag
```

Comment:
Hmm I have to insist that IMO "Shutdown Flag" would lead me to think that 1 means the system is shutdown.
If all the rest still prefers it I'll not oppose, but it's kind of an important param to make people confused about how it is used.

Reply:
Yeah, you have a good point...I don't find "Shutdown Flag" confusing because I'm already familiar with the codebase, but for someone who isn't, it may be a different story.

Reply:
On second thought, I think "Access Flag" is more accurate.

Reply:
"Access Flag" is the only description I am categorically opposed to.

I think "Active Flag" or maybe "Operational Flag" or even just "Liveness Flag" are all better than "Access Flag".

No comment is also a better option than "Access Flag".

Reply:
I like "Active Flag" or "Liveness Flag".

Reply:
Going with `Active Flag`